_includes/code/install/embedded.py

```diff
@@ -22,19 +28,26 @@
 client = weaviate.WeaviateClient(
     embedded_options=EmbeddedOptions(
         additional_env_vars={
-        "ENABLE_MODULES":
-        "backup-s3,text2vec-openai,text2vec-cohere,text2vec-huggingface,ref2vec-centroid,generative-openai,qna-openai"}
+            "ENABLE_MODULES": "backup-filesystem,text2vec-openai,text2vec-cohere,text2vec-huggingface,ref2vec-centroid,generative-openai,qna-openai",
+            "BACKUP_FILESYSTEM_PATH": "/tmp/backups"
+        }
     )
     # Add additional options here. For syntax, see the Python client documentation.
 )
 
+client.connect()  # If instantiating `WeaviateClient` directly, you must call `connect()` to connect to the server.
```

Comment:
Is this needed? Doesn't ln 7 make the connection?

Reply:
7 does, but I think that `client` object is a different one to that instantiated at line 28, as the `WeaviateClient` class is instantiated to a new client object.

The script fails for me when I try to run it without `client.connect()`.